fsharp-backend/tests/Tests/LibExecution.Tests.fs

```diff
@@ -441,38 +504,48 @@ let fileTests () : Test =
         currentFn <- { currentFn with code = currentFn.code + "\n" + line }
       // 1-line test
       | Regex @"^(.*)\s+//\s+(.*)$" [ code; comment ] ->
-        let initializeCanvas = filename = "internal.tests" || currentGroup.dbs <> []
         let test =
           t
             owner
             initializeCanvas
+            (Randomized $"{comment} (line {i})")
             $"{comment} (line {i})"
             code
-            currentGroup.dbs
+            currentGroup.extras.dbs
             packageFunctions
             functions
-            currentTest.workers
+            currentTest.extras.workers
+            currentTest.extras.staticAssetsDeployHashes
 
         currentGroup <- { currentGroup with tests = currentGroup.tests @ [ test ] }
       | Regex @"^(.*)\s*$" [ code ] ->
-        let initializeCanvas = filename = "internal.tests" || currentGroup.dbs <> []
         let test =
           t
             owner
             initializeCanvas
+            (Randomized $"line {i}")
             $"line {i}"
             code
-            currentGroup.dbs
+            currentGroup.extras.dbs
             packageFunctions
             functions
-            currentTest.workers
+            currentTest.extras.workers
+            currentTest.extras.staticAssetsDeployHashes
 
         currentGroup <- { currentGroup with tests = currentGroup.tests @ [ test ] }
 
       | _ -> raise (System.Exception $"can't parse line {i}: {line}"))
 
     finish ()
-    testList $"Tests from {filename}" allTests)
+    let tests = testList $"Tests from {filename}" fileTests
+    // Staticassets.tests use a combination of ExactCanvasName and
+    // StaticAssetsDeployHash, which can be racy. See comment in staticassets.tests.
```

Comment:
replace 'racy' with less ambiguous term like "Race conditions can cause problems with these tests" here and in `staticassets.tests`?

Wasn't sure if 'racy' meant 'involves race conditions' or 'annoying/dicy' or something else, at first read